### PR TITLE
Accounts: Add caveats about consumption

### PR DESCRIPTION
**Pull request by rhetoric101**

### CODE REVIEW

**Comment by paperclypse** [comment]
@rhetoric101 I missed this one. Do you still need a review/approval on it?

**Comment by rhetoric101** [comment]
@paperclypse I am still waiting on an SME for approval.

**Comment by paperclypse** [comment]
@rhetoric101 Let's close this one for now and re-open it when the SME gets back to you.

**Comment by rhetoric101** [comment]
@paperclypse I received approval from the SMEs. Could you review and approve it?

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts.mdx
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts.mdx:31]:
```suggestion
* `NrConsumption`: Use this to dive deep into the details of your usage hour-by-hour. It's less helpful as a proxy for your bill.
```

How about this?

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts.mdx
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts.mdx:32]:
```suggestion
* `NrMTDConsumption`: Use this to see a good approximation of your monthly aggregated usage.
```

How about this?

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx:38]:
```suggestion
To find the usage UI, go to **[one.newrelic.com](https://one.newrelic.com)**, click the [user menu](/docs/accounts/accounts-billing/general-account-settings/intro-account-settings), and then click **Administration**. The **Plan & usage** UI page shows an overview of your usage like this:
```

How about this?

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts.mdx
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts.mdx:24]:
```suggestion
* Use these tools to find and track your areas of high usage.
```

**Review by paperclypse — APPROVED** [review]
I left some comments for you to consider.

**Review by paperclypse — APPROVED** [review]
Looks good!

**Pull request merged**
